### PR TITLE
Drop and create a table on each test run

### PR DESCRIPTION
**Pull request by hudo**
MS changed exception message and removed "Json" prefix so we were not creating a new table successfully. 
Test case updated to drop the table after each run, we will be able to see now if this starts failing again if exception message gets changed again.

### CODE REVIEW

File: src/Tests/Eshopworld.Telemetry.Tests/BigBrotherUseKustoTest.cs [hunk 1]
```diff
@@ -72,6 +76,11 @@ await Policy.Handle<Exception>()
 
                 reader.Read().Should().BeTrue();
                 reader.GetInt64(0).Should().Be(1);
+
+                await KustoAdminClient.ExecuteControlCommandAsync(
```
Comment (hudo) [inline src/Tests/Eshopworld.Telemetry.Tests/BigBrotherUseKustoTest.cs]: I will refactor this because @arturek just gave me a speech about this test not beeing consistent and depends on the previous run. He's right unfortunately

Comment (dguerin) [reply src/Tests/Eshopworld.Telemetry.Tests/BigBrotherUseKustoTest.cs]: yeah this happens some times. I find if you don't add @arturek to the PR, it gets merged a lot quicker :D

Comment (arturek) [reply src/Tests/Eshopworld.Telemetry.Tests/BigBrotherUseKustoTest.cs]: I'm sorry to cause troubles. Our initial discussion was only about a very simple one-line fix. IMHO adding a test in this case is optional (it's a good thing to do if there's none but it isn't a requirement). In my view the proposed change to the test code doesn't improve it (mentioned dependency is one of the issues). So if nobody objects I would suggest to return to the initial idea and only keep modification of KustoExtentions.cs in this PR. It will resolve the problem without introducing others.

Comment (hudo) [reply src/Tests/Eshopworld.Telemetry.Tests/BigBrotherUseKustoTest.cs]: @arturek why do you think this test can introduce problems?

**Comment by djfr** [comment]
So much hassle for such a tiny and short lived change 😩  I will redo all the error checking code, including this, when I integrate telemetry.

@hudo You haven't bumped package version, you should before releasing this into master, including release notes.

**Comment by dguerin** [comment]
@hudo @djfr there is a nuget bump to v6 of the kusto client that might have some interesting updates.

**Comment by djfr** [comment]
That [has been done in kusto/master](https://github.com/eShopWorld/telemetry/commit/925d71115420bf1908f17d4626e84555e0513c6c) already. Avoid doing the same thing in concurrent PRs.

[This commit](https://github.com/eShopWorld/telemetry/commit/18a8c44393cbd95891804730abef5fe6a5476feb) inline for the next PR in the Kusto improvements also voids this PR, since tables and mappings are queried first before attempting to create the table or mapping.
